pandas/core/reshape/tile.py

```diff
@@ -226,7 +226,10 @@ def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
             raise ValueError('Overlapping IntervalIndex is not accepted.')
 
     else:
-        bins = np.asarray(bins)
+        if is_datetime64tz_dtype(bins):
```

Comment:
? ``.to_numpy()`` is not enough here

Reply:
We could, though we'll still need the test for datetimetz so that we don't end up with an object-dtype ndarray.